src/epiviz.gl/webgl-vis.js

```diff
@@ -31,6 +31,7 @@ class WebGLVis {
       "viewport",
       "currentXRange",
       "currentYRange",
+      "uniDirectionalSelectionEnabled",
```

Comment:
no hold on, what I meant was, `uniDirectionalSelectionEnabled` should be replaced with `lockedX` and `lockedY`. 

if `lockedX` and `box` is the tool =>  x-axis is locked, hence all selections behave like `boxh`
similarly for `lockedY` and `boxv`

Reply:
I got it now. I can do the thing you suggested, But It will change the behavior completely. With this code. When you move horizontally or vertically within 30px buffer, It automatically switches to horizontal and vertical select.

If I make it in sync with lockedX and lockedY the interaction behaviour will be different. In that case we will first have to lock the axis in order to select in one direction. Then there is one disadvantage as well. What if user wants to lock x axis for zoom purpose only but still wants to do a box select of smaller area. That won't be possible because it will do unidirectional select only in that case.